tests/unit_tests/data_science_utils/schema/test_column.py

```diff
@@ -1,119 +1,264 @@
+import random
+import string
+from dataclasses import FrozenInstanceError
+from typing import (
+    Any,
+    Iterator,
+)
+
 import pytest
 from typeguard import TypeCheckError
 
 from exasol.analytics.schema import (
+    BooleanColumn,
+    CharColumn,
+    CharSet,
     Column,
-    ColumnNameBuilder,
-    ColumnType,
-    SizeUnit,
-    decimal_column,
-    hashtype_column,
-    timestamp_column,
-    varchar_column,
+    ColumnName,
+    DateColumn,
+    DecimalColumn,
+    DoublePrecisionColumn,
+    GeometryColumn,
+    HashSizeUnit,
+    HashTypeColumn,
+    TimeStampColumn,
+    UnsupportedSqlType,
+    VarCharColumn,
+)
+from exasol.analytics.schema.column_types import (
+    PyexasolOption,
+    SqlType,
 )
 
+TEST_CASES = [
```

Comment:
Can you define a dataclass for the test case? It will make the code easier to read.

Reply:
I don't understand the question.
The purpose of the variable TEST_CASES is to reuse the same list of test cases in multiple actual test cases:
* test_from_sql_spec
* test_for_create